services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/BaseSpecification.groovy

```diff
@@ -50,24 +47,25 @@ class BaseSpecification extends SpringSpecification implements SetupOnce {
     }
 
     @HealthCheck
-    def "Given hardware topology is suitable for running the test suite"() {
-        requireProfiles("hardware")
+    def "Kilda is UP and network topology is clean"() {
+        expect: "Kilda's health check request is successful"
+        northbound.getHealthCheck().components["kafka"] == "operational"
 
-        expect: "Topology and overall kilda state are suitable for running further test suite"
+        and: "All links are active. No flows and link props are present"
         def links = northbound.getAllLinks()
         verifyAll {
-            northbound.activeSwitches.size() == topologyDefinition.activeSwitches.size()
-            northbound.allFlows.empty
-            northbound.allLinkProps.empty
             links.findAll {
                 it.state == IslChangeType.DISCOVERED
             }.size() == topologyDefinition.islsForActiveSwitches.size() * 2
             links.findAll { it.state == IslChangeType.FAILED }.empty
+            links.findAll { it.availableBandwidth != it.speed }.empty
```

Comment:
This verification was intentionally put to the 'seconds phase of verifications', because this is a consequence of having flows in the system. My thought was, that it doesn't make sense to inform about bw!=speed if there a flows in the system (same about rules) -> second phase should be reported only after 'basic' things like flows in the system are fixed